src/towncrier/_settings/load.py

```diff
@@ -55,6 +55,7 @@ class Config:
     create_eof_newline: bool = True
     create_add_extension: bool = True
     ignore: list[str] | None = None
+    fragment_filename_stem_pattern: str | None = None
```

Comment:
we could find it a shorter name

Reply:
My suggetion is `issue_pattern` 

Inside the code and the documentation, we should refer to this as the "issue ID" , or just "issue" ... it used to be called "ticket" but it was recently renamed to "issue"

Reply:
Just asking ... can we have this as just `str` ... with default value as empty string.

It should simplify the typing for this variable.